docs/user-guide/cli-using-daemon-mode.md

```diff
@@ -1,46 +1,61 @@
 
-You can download and install the zowe native executable from Zowe Downloads (preferred) or from the Zowe Github repository. To install the zowe native executable from Zowe Downloads, see [Install Zowe CLI from a download](cli-install-cli-next.md#install-zowe-cli-from-a-download).
+## Running Zowe commands in daemon mode
 
-To download and install the zowe native executable from the Zowe Github repository:
+When you run Zowe in daemon mode, you run all Zowe commands as you normally run them. The first time you run a command, it starts the daemon in the background automatically and runs your desired Zowe command. Since the first Zowe command starts the daemon, the first command usually runs slower than a traditional Zowe command. However, subsequent Zowe commands run significantly faster. The daemon continues to run in the background until you close your terminal window.
 
-1. Navigate to the Zowe CLI GitHub [repository](https://github.com/zowe/zowe-cli)
-2. Under the **Release** section, click **Native Client Release**.
-   The Download page is displayed.
-3. Click the link to download the executable (.tgz file) for your operating system.
-4. Unzip the zowe .tgz file and place the zowe (or zowe.exe) file into a directory which occurs on your PATH earlier than the directory containing the NodeJS zowe command.
+If you want to run Zowe in daemon mode constantly, add the path of the $ZOWE_CLI_HOME/bin (.zowe/bin) executables to the directory path of your computer's environment variables. When you configure Zowe to constantly run in daemon mode, daemon mode starts automatically after you issue the first Zowe command. When you open multiple terminal windows and issue Zowe commands, the Zowe commands in all of the terminal windows run in daemon mode.
 
-**Note:** The Mac executable may need an exception for an unidentified developer: [Open a Mac app from an unidentified developer](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac)
+### Enable daemon mode
 
-## Running zowe commands with the zowe executable
+The following steps describe how to enable daemon mode and how to (optionally) configure Zowe to run in daemon mode constantly.
 
-Verify that you have installed [the @next release of Zowe CLI](cli-install-cli-next.md) and the [zowe native executable](#install-the-zowex-native-executable).
+1. Open a terminal window and issue the following command:
+   ```
+   zowe daemon enable
+   ```
+   The command copies the Zowe executables for your operating system into the $ZOWE_CLI_HOME/bin (.zowe/bin) directory. The next command that you issue starts the daemon.
+2. (Optional) Add the path to the Zowe executables to the environment variables for your computer. For example:
```

Comment:
@gejohnston is this optional? I thought it had to be added to the path for daemon mode to work?

Reply:
You are correct Mike. I overlooked the 'optional' remark. The 'optional' remark should be removed.